src/main/java/chattime/TaskList.java

```diff
@@ -0,0 +1,58 @@
+package chattime;
+
+import java.util.ArrayList;
+
+import chattime.task.Task;
+
+/**
+ * Represents a list object to store user's tasks.
+ *
+ */
+public class TaskList {
+    private ArrayList<Task> tasks;
+
+    /**
+     * Constructs a taskList object with ArrayList as data structure.
+     *
+     * @param initialTasks Stored data loaded from storage.
+     */
+    public TaskList(ArrayList<Task> initialTasks) {
+        tasks = initialTasks;
+    }
+
+    /**
+     * Adds a new task to current task list.
+     * @param task New task to be added.
+     */
+    public void addTask(Task task) {
+        tasks.add(task);
+    }
+
+    /**
+     * Removes index-th task object from current task list.
+     *
+     * @param index Input index from user, index-th task in task list to be removed.
+     */
+    public void removeListMember(int index) {
```

Comment:
As the ArrayList has been defined to contain objects of type Task, maybe consider changing this method name to removeTask instead?

Reply:
Thanks for your suggestion, it looks better now!